src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -113,7 +109,7 @@ public void quit() {
     sendCommand(QUIT);
   }
 
-  public void exists(final byte[] key) {
+  public void exists(final byte[]... key) {
```

Comment:
@charsyam 
We're breaking binary compatibility now. 
Could you leave origin method as is but adding @Deprecated?